src/main/java/co/rsk/federate/config/PowpegNodeConfigParameter.java

```diff
@@ -0,0 +1,51 @@
+package co.rsk.federate.config;
+
+import java.util.Objects;
+import java.util.function.Function;
+
+public enum PowpegNodeConfigParameter {
+  FEDERATOR_ENABLED("federator.enabled", Boolean.toString(false)),
+  PEGOUT_ENABLED("federator.pegout.enabled", Boolean.toString(false)),
+  UPDATE_BRIDGE_TIMER_ENABLED("federator.updateBridgeTimerEnabled", Boolean.toString(true)),
+  UPDATE_BRIDGE_BTC_BLOCKCHAIN("federator.updateBridgeBtcBlockchain", Boolean.toString(false)),
+  UPDATE_BRIDGE_BTC_COINBASE_TRANSACTIONS("federator.updateBridgeBtcCoinbaseTransactions", Boolean.toString(false)),
+  UPDATE_BRIDGE_BTC_TRANSACTIONS("federator.updateBridgeBtcTransactions", Boolean.toString(false)),
```

Comment:
Let's make these true by default. We want them false by default now, but for the next version we want them true.

But let's do these changes on a separate task, to keep things organized

Reply:
Okay, I have created a new task for this.